packages/gui/index.ts

```diff
@@ -6,6 +6,7 @@ export * from './components/Select';
 export * from './components/Slider';
 export * from './components/Text';
 export * from './components/View';
+export * from './hooks';
```

Comment:
anyway, I think hooks is more related to the domain of `@walless/app` instead of `@walless/gui`.